Sorting/bubble.cpp

```diff
@@ -16,9 +17,6 @@ void Sort::BubbleSort(){
 
     f(i, sorted.size() - 1)
         f(j, sorted.size() - 1)
-            if(sorted[j] > sorted[j + 1]){
-                int temp = sorted[j];
-                sorted[j] = sorted[j+1];
-                sorted[j+1] = temp; 
-            }
+            if(sorted[j] > sorted[j + 1])
+                std::swap(sorted[j], sorted[j + 1]);
```

Comment:
Please revert. You may do this in a separate PR. This goes as type 2. Please go through README.md, you can't perform two tasks in one PR.